### ISSUE TITLE
util fails to import in Python3

### ISSUE THREAD

**Issue by CrepeGoat** [MideTechnology/ebmlite#36]
```pycon
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ebmlite.util
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/crepegoat/repos/MideTechnology/cloud-tools/venv/lib/python3.6/site-packages/ebmlite/util.py", line 25, in <module>
    import core as core
ModuleNotFoundError: No module named 'core'
```

**Comment by CrepeGoat** [comment]
changed util.py to fix two import statements, seemed to work after but did not extensively test:

before:
```python
import ebmlite.core as core
import ebmlite.encoding as encoding
```

after:
```python
from . import core
from . import encoding
```

**Comment by CFlaniganMide** [comment]
Updating this again because it became relevant again.  I'm not 100% sure what the best way to do imports is.  If this is a part of the ebmlite module, then it should use relative imports.  If it's a command line script, it needs to import an installed ebmlite instance.  Likely the solution is two separate files, but we should plan that a bit more.

**Comment by StokesMIDE** [comment]
Yeah, the library needs to be separated from the script, particularly if we're going to put the script with the other Python utilities (see #44).

**Comment by CFlaniganMide** [comment]
I think we need to do some refactoring, but I'll put that in #44 because I think that's the most relevant one.  For this, I think we should assume that the utils.py file that lives in the ebmlite directory and ships as part of the module should be importing the same as the rest of ebmlite.

**Comment by StokesMIDE** [comment]
Agreed. The imports are wonky because it can work as a stand-alone script, so it couldn't use relative imports. I think we need to break things and make `util.py` a regular library, and implement a utility (`ebmlutil`) or utilities (`xml2ebml`, `ebml2xml`, `ebmlview`). Users will have to adjust their workflow, but it won't hurt code that uses `util.py`. Humans are more adaptable, anyway.

**Comment by StokesMIDE** [comment]
I believe this has been fixed in PR #46.
